src/spatialmedia/box.cpp

```diff
@@ -1,19 +1,19 @@
 /*****************************************************************************
```

Comment:
I can add an exception for the spatialmedia directory if we want.

Reply:
Yes, let's exclude spatialmedia as well.

Reply:
Done